go/vt/vttablet/tabletserver/vstreamer/vstreamer.go

```diff
@@ -169,8 +169,10 @@ func (vs *vstreamer) Stream() error {
 
 	// Ensure sh is Open. If vttablet came up in a non_serving role,
 	// the schema engine may not have been initialized.
-	if err := vs.sh.Open(); err != nil {
-		return wrapError(err, vs.pos)
+	if !vs.se.SkipMetaCheck {
```

Comment:
why did we need this?

Reply:
for getting GTID from timestamp, we don't need the query to the information_schema tablet meta information fetch and with ripple, this query is not supported

Reply:
ah ok.